src/spinegeneric/cli/generate_figure.py

```diff
@@ -1343,7 +1340,3 @@ def main(argv=sys.argv[1:]):
     if args.output_html:
         # Generate interactive html T1w vs. T2w figure
         generate_figure_t1_t2_plotly(df, csa_t1, csa_t2)
-
-
-if __name__ == "__main__":
-    main()
```

Comment:
Regarding 

> * Remove `if __name__ == "__main__":` footers
>  (The CLI scripts are intended to be called via the pip-installed entry
>  points `sg_something`, rather than `python -m spinegeneric.something`.)

I assume the only reason SCT needs the `__name__` footers is because our entry points call a launcher script which then calls the `sct_` scripts using a subprocess? Whereas here, the `sg_` entry points tap directly into `main()`, making it unnecessary...

Reply:
That's right!